dart/lib/src/protocol/sentry_level.dart

```diff
@@ -14,4 +14,30 @@ class SentryLevel {
   /// API name of the level as it is encoded in the JSON protocol.
   final String name;
   final int ordinal;
+
+  /// For use with Dart's
+  /// [`log`](https://api.dart.dev/stable/2.12.4/dart-developer/log.html)
+  /// function.
+  /// These levels are inspired by
+  /// https://pub.dev/documentation/logging/latest/logging/Level-class.html
+  int toLogLevel() {
+    switch (this) {
+      // Level.SHOUT
+      case SentryLevel.fatal:
+        return 1200;
+      // Level.SEVERE
+      case SentryLevel.error:
+        return 1000;
+      // Level.SEVERE
+      case SentryLevel.warning:
+        return 900;
+      // Level.INFO
+      case SentryLevel.info:
+        return 800;
+      // Level.CONFIG
+      case SentryLevel.debug:
+        return 700;
+    }
+    return 1200;
```

Comment:
I think the default should be always equivalent to `SentryLevel.debug`

Reply:
```suggestion
    return 700;
```